scripts/ci_test_build.sh

```diff
@@ -8,31 +8,13 @@ set -e
 # Treat warnings as errors.
 if [ "$TRAVIS_COMPILER" = "clang" ]; then
     COMPILER_OPTIONS+=" -Werror"
```

Comment:
Nit: $TRAVIS_COMPILER should always be clang for this file and $TRAVIS_OS_NAME will always be "linux" since we're removing the ports. Might be neater to just remove the ifs.

Reply:
It will be easier to add back in the ports if these stay untouched.